modules/gridss/1.1/gridss.smk

```diff
@@ -493,31 +452,17 @@ rule _gridss_gripss_to_bedpe:
         op.as_one_line("""
         zcat {input.vcf} | 
             awk '$1 ~ /^#/ || $5 ~ /:/' | 
-            svtools vcftobedpe -o {output.bedpe}
+            svtools vcftobedpe | grep -v "##" > {output.bedpe}
         """)
      
 
 # Symlink the final output files into the module results directory (under '99-outputs/')
-rule _gridss_output_viral_vcf:
-    input:
-        vcf = str(rules._gridss_unpaired_filter.output.vcf), 
-        tbi = str(rules._gridss_unpaired_filter.output.tbi), 
-        bedpe = str(rules._gridss_unpaired_to_bedpe.output.bedpe)
-    output:
-        vcf = CFG["dirs"]["outputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss_viral_annotation_filtered.vcf.gz", 
-        tbi = CFG["dirs"]["outputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss_viral_annotation_filtered.vcf.gz.tbi", 
-        bedpe = CFG["dirs"]["outputs"] + "bedpe/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss_viral_annotation_filtered.bedpe"
-    run:
-        op.relative_symlink(input.vcf, output.vcf, in_module=True)
-        op.relative_symlink(input.tbi, output.tbi, in_module=True)
-        op.relative_symlink(input.bedpe, output.bedpe, in_module=True)
-
 rule _gridss_output_somatic_vcf:
     input:
         filtered = str(rules._gridss_filter_gripss.output.vcf), 
         filtered_tbi = str(rules._gridss_filter_gripss.output.tbi), 
         somatic = str(rules._gridss_run_gripss.output.vcf), 
-        somatic_tbi = str(rules._gridss_run_gripss.output.tbi),
+        somatic_tbi = str(rules._gridss_run_gripss.output.vcf),
```

Comment:
Does it seem that this input (`somatic_tbi`) is referring to the same file as `somatic` ?